labelbox/data/metrics/group.py

```diff
@@ -158,3 +157,19 @@ def _create_feature_lookup(features: List[FeatureSchema],
     for feature in features:
         grouped_features[getattr(feature, key)].append(feature)
     return grouped_features
+
+
+def has_no_matching_annotations(ground_truths: List[ObjectAnnotation],
```

Comment:
what is a `matching` annotation? wouldnt a matching prediction and label have the same schemaId? Would it make sense to check for that in this function, or is it not that simple?
Would it make sense to make sure the lengths are the same?

Reply:
All annotations passed to this function have the same schema ids. So this function checks if we even need to run the metric calculation code - if there is no ground truth or no prediction then we just give a default score.